### ISSUE TITLE
Problemen spotify website- Album foto's grootes etc.

### ISSUE THREAD

**Issue by Serdemcann62** [zavo-web/spotify-web-replica-Serdemcann62#1]
Beste Mevrouw,

Zoals u kunt zien in mijn website, heb ik wat verder gewerkt, maar ik heb enkele problemen.

- Foto's van de album's zijn niet juist, en ik kan zij niet aanpassen.

Kunt u me hiervoor verder helpen aub?
En wat moeten wij nog doen voor de examen dan de website?

Alvast bedankt
Mvg
Serdem

**Comment by sarabarrerariano-zavo** [comment]
Hoe bedoel je dat je dat niet kan aanpassen?

**Comment by sarabarrerariano-zavo** [comment]
Je gebruikt altijd dezelfde afbeelding source, die van billie

**Comment by sarabarrerariano-zavo** [comment]
Examens: daar komt maandag het laatste voor op smartschool. Maar eigenlijk staat alles er op. Op het examen geef je een presentatie van alle fases obv je analyse.

**Comment by Serdemcann62** [comment]
> Je gebruikt altijd dezelfde afbeelding source, die van billie



> Je gebruikt altijd dezelfde afbeelding source, die van billie

Ja, maar de groottes van de foto’s kan ik niet aanpassen om die zoals in de website van spotify te doen.

**Comment by sarabarrerariano-zavo** [comment]
.album-grid img {
    width: 120px;
    border-radius: 15px;
}

je afbeeldingen hebben een max-width, dat mag niet om uw img staan, maar moet op de parent van het kaartje staan, en img geef je width: 100%

**Comment by sarabarrerariano-zavo** [comment]
<img width="845" alt="image" src="https://github.com/user-attachments/assets/d49cdef7-156a-44aa-9497-62743a2cbb94">

**Comment by Serdemcann62** [comment]
Ja, maar als ik deze aanpassingen doe, krijg ik niks? waarom zou dat komen?

**Comment by sarabarrerariano-zavo** [comment]
Omdat sommige css eigenschappen zoals width worden overgeërfd van de parents. Dus elk kind gaat width 100% nodig hebben. En dan kan je de hoogste parent een max width geven of de breedte dat je wil.

**Comment by Serdemcann62** [comment]
Ik heb nu nog is gepusht; ik heb de aanpassingen gedaan dat u zei, maar ik krijg het nog niet, is er iets anders fout misschien?
